tradeit-android-sdk/src/main/java/trade/it/android/sdk/model/TradeItErrorResult.java

```diff
@@ -0,0 +1,31 @@
+package trade.it.android.sdk.model;
+
+
+import java.util.Arrays;
+import java.util.List;
+
+import it.trade.tradeitapi.model.TradeItErrorCode;
+
+public class TradeItErrorResult {
+    private TradeItErrorCode errorCode = null;
+    private String shortMessage = null;
+    private List<String> longMessages = Arrays.asList("Trading is temporarily unavailable. Please try again in a few minutes.");
```

Comment:
You can use singletonList here.  We should extract messages that will show up on the UI into resources, in case we need translations later.